Editor/Resources/EditorWindow/Pages/AwsUserProfilesPage.uxml

```diff
@@ -39,15 +40,18 @@
             <ui:VisualElement class="separator separator--horizontal separator--centered">
                 <ui:Button name="UserProfilePageBootstrapStartButton" class="button button--primary" text="Bootstrap Profile"/>
                 <ui:Button name="UserProfilePageBootstrapAnotherProfileButton" class="button" text="Add another Profile"/>
-                <ui:Label name="UserProfilePageBootstrapHelpLink" class="link" text="What is bootstrapping?  \uf08e"/>
+                <ui:VisualElement name="UserProfilePageBootstrapHelpLink" class="external-link float-right">
+                    <ui:Label text="What is bootstrapping?"/>
```

Comment:
This needs to come from the text provider and shouldn't be hardcoded

Reply:
Also looking at `Strings.cs` I don't see the suffix `LinkText` used anywhere. Can we review how other links are structured and follow the same conventions?

Reply:
Solved both issues.